docs/rules/0192/deprecated-method-comment.md

```diff
@@ -0,0 +1,65 @@
+--------------------------------------------------------------------------------
+
+rule: aip: 192 name: [core, '0192', deprecated-method-comment] summary:
+Deprecated methods must have a corresponding comment. permalink:
+/192/deprecated-method-comment redirect_from:
```

Comment:
This should redirect from `/0192/deprecated-method-comment`.

Reply:
Done.